BlueprintUI/Sources/Layout/GridRow.swift

```diff
@@ -120,6 +120,52 @@ extension GridRow {
             return attributes
         }
 
+        private func items(subelements: Subelements) -> [(traits: Traits, content: Measurable)] {
+            subelements.map { subelement in
+                let traits = subelement.gridRowLayoutTraits
+                let measurable = Measurer { constraint in
+                    subelement.sizeThatFits(constraint)
+                }
+                return (traits, measurable)
+            }
+        }
+
+        func sizeThatFits(proposal: SizeConstraint, subelements: Subelements, cache: inout Cache) -> CGSize {
+            guard subelements.count > 0 else {
+                return .zero
+            }
+
+            let items = items(subelements: subelements)
+
+            let frames = _frames(in: proposal, items: items)
+
+            // Measure the the row to be as wide as the sum of its children and as tall as its tallest child.
+            let size = frames.reduce(.zero) { size, frame in
+                CGSize(width: frame.maxX, height: max(size.height, frame.height))
+            }
```

Comment:
I was expecting `width: max(size.width, frame.maxX)`.

Does the `_frames` call guarantee that the right-most frame will be the last one?

(aside: I guess we could keep track of the max height within `_frames` and return it since we're already iterating over them there. Then the size calculation avoids iterating: `let size = CGSize(width: value.frames.last!.maxX, height: value.maxHeight)`. Assuming we're comfortable with relying on the last frame for width.)